packages/design-preview/src/components/style-variation.tsx

```diff
@@ -2,17 +2,24 @@ import { GlobalStylesContext } from '@wordpress/edit-site/build-module/component
 import { mergeBaseAndUserConfigs } from '@wordpress/edit-site/build-module/components/global-styles/global-styles-provider';
 import Preview from '@wordpress/edit-site/build-module/components/global-styles/preview';
 import { useMemo } from '@wordpress/element';
-import type { StyleVariation } from '@automattic/design-picker';
+import classnames from 'classnames';
+import type { StyleVariation } from '@automattic/design-picker/src/types';
 import './style.scss';
 
+const SPACE_BAR_KEYCODE = 32;
+
 interface StyleVariationPreviewProps {
 	variation: StyleVariation;
 	base?: StyleVariation;
+	isActive?: boolean;
```

Comment:
[nit] I think both of the new props (and even `base`) can be required (not optional) to simplify the code?

Reply:
Updated!